### ISSUE TITLE
[Markup and meta data] HTML markup isn't valid. (09.01.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#34873]
#### VFS team name
Claims & Appeals

#### VFS product name
HLR V2

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

---

### Platform Issue
HTML markup isn't valid.

### Issue Details
On the /start page, the "Disability compensation claim" radio button has an aria-describedby attribute pointing to an ID that doesn't exist on the page.

### VA.gov Experience Standard
[Category Number 09, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A

---

### Platform Recommendation
Either remove the invalid aria-describedby attribute, or add an appropriate element with ID "legacy-choice".

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by Mottie** [comment]
Fixed. It should be available in staging soon.

**Comment by briandeconinck** [comment]
Re-tested and confirm that the `aria-describedby` attribute has been removed.
